db/seeders/correspondence_type_seeder.rb

```diff
@@ -37,7 +37,7 @@ def seed!
 
     rec = CorrespondenceType.find_by(abbreviation: 'OFFENDER_SAR')
     rec = CorrespondenceType.new if rec.nil?
-    rec.update!(name: 'Offender SAR (OFFENDER)',
+    rec.update!(name: 'Offender subject access request',
```

Comment:
Should this be `Offender Subject Access Request` ?

Reply:
Just copied what's on the ticket. I guess we can check this with Alex though as you might be right here.

Reply:
I confirmed with Alex who suggested to keep as is since it's consistent with other case types: 
![image](https://user-images.githubusercontent.com/22935203/64104002-47b8d100-cd6b-11e9-86ad-eb1cd49fa59a.png)